src/components/Conference.tsx

```diff
@@ -142,15 +142,18 @@ export class Conference extends React.Component<IConferenceProps, IConferenceSta
 
     private renderStream(stream: ConferenceStream) {
         let className = 'rcw-remote-stream';
+        let muted;
         if (stream.local) {
```

Comment:
Yes, local stream should be muted by this. This is for the default render experience if user doesn't provide a ConferenceRenderer function.